buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -44,7 +44,7 @@ object Dependencies {
         }
 
         object Compose {
-            const val version = "1.1.1"
+            const val version = "1.2.0-alpha07"
```

Comment:
Shall we update to `beta-01` (or the latest Compose library when the tests pass)?